phpBB/install/schemas/schema_data.sql

```diff
@@ -835,12 +836,16 @@ INSERT INTO phpbb_extensions (group_id, extension) VALUES (4, 'ods');
 INSERT INTO phpbb_extensions (group_id, extension) VALUES (4, 'odt');
 INSERT INTO phpbb_extensions (group_id, extension) VALUES (4, 'rtf');
 
-INSERT INTO phpbb_extensions (group_id, extension) VALUES (5, 'mp3');
 INSERT INTO phpbb_extensions (group_id, extension) VALUES (5, 'mpeg');
 INSERT INTO phpbb_extensions (group_id, extension) VALUES (5, 'mpg');
-INSERT INTO phpbb_extensions (group_id, extension) VALUES (5, 'ogg');
 INSERT INTO phpbb_extensions (group_id, extension) VALUES (5, 'ogm');
 
+INSERT INTO phpbb_extensions (group_id, extension) VALUES (6, 'mp3');
+INSERT INTO phpbb_extensions (group_id, extension) VALUES (6, 'wav');
+INSERT INTO phpbb_extensions (group_id, extension) VALUES (6, 'm4a');
+INSERT INTO phpbb_extensions (group_id, extension) VALUES (6, 'ogg');
+INSERT INTO phpbb_extensions (group_id, extension) VALUES (6, 'webm');
```

Comment:
For this and above changes in this file - if you change it for new installations, then migration for update existing ones is required.

Reply:
No migration was done for `phpbb_extensions` before, Users are able to change this values, So we can't expect what is in user table

Reply:
The fact extensions table wasn't changed previously doesn't mean it shouldn't be changed :) 
Without migration, updated boards will be simply missing the feature.